README.md

```diff
@@ -55,3 +55,41 @@ For primitive arrays, define the Views the same way but without the source prop
      <LongTextInput />
  </EmbeddedArrayInput>
 ```
+
+Uses of custom action buttons
+
+```jsx
+ import FlatButton from 'material-ui/FlatButton';
+ import ActionDeleteIcon from 'material-ui/svg-icons/action/delete';
+ const CustomDeleteButton = ({items, index, removeItem}) => (
```

Comment:
What is `removeItem` here ? and How do we access it. As far as i saw somewhere else, you're passing only `items` and `index` to the this function.

Reply:
My mistake... Fix it soon

Reply:
My mistake... Fix it soon